src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -122,7 +122,7 @@ public string InstanceId
             {
                 if (_instanceId == null)
                 {
-                    _instanceId = _settingsManager.InstanceId;
+                    _instanceId = Guid.NewGuid().ToString();
```

Comment:
Why did this change?

Reply:
_settingsManager.InstanceId is static value and is used only once in this file to get the lease on blob.